src/lib.rs

```diff
@@ -51,11 +50,12 @@ use core::sync::atomic::{AtomicBool, AtomicPtr};
 use futures_core::stream::{FusedStream, Stream};
 use futures_core::task::{Context, Poll};
 use futures_util::task::{ArcWake, AtomicWaker};
+use std::marker::PhantomData;
```

Comment:
Ah, this needs to stay `core` for `no_std`-compatibility :+1:

Reply:
Huh, I have no idea why this changed.